pennylane/transforms/sign_expand.py

```diff
@@ -0,0 +1,359 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Contains the sign (and xi) decomposition tape transform, implementation of ideas from arXiv:2207.09479
+"""
+# pylint: disable=protected-access
+import json
+from os import path
+import pennylane as qml
+from pennylane import numpy as np
+from .batch_transform import batch_transform
+
+
+def ControlledPauliEvolution(theta, wires, pauli_word, controls):
+    r"""Controlled Evolution under generic Pauli words, adapted from the decomposition of
+    qml.PauliRot to suit our needs
+
+
+    Args:
+        theta (float): rotation angle :math:`\theta`
+        pauli_word (string): the Pauli word defining the rotation
+        wires (Iterable, Wires): the wires the operation acts on
+        controls (List[control1, control2]): The two additional controls to implement the
+          Hadamard test and the quantum signal processing part on
+
+    Returns:
+        list[Operator]: decomposition that make up the controlled evolution
+    """
+    active_wires, active_gates = zip(
+        *[(wire, gate) for wire, gate in zip(wires, pauli_word) if gate != "I"]
+    )
+
+    ops = []
+    for wire, gate in zip(active_wires, active_gates):
+        if gate == "X":
+            ops.append(qml.Hadamard(wires=[wire]))
+        elif gate == "Y":
+            ops.append(qml.RX(np.pi / 2, wires=[wire]))
+
+    ops.append(qml.CNOT(wires=[controls[1], wires[0]]))
+    ops.extend(MultiCRZ(theta, wires=list(active_wires), control=controls[0]))
```

Comment:
I just tried to use `qml.ctrl` locally and all the tests in `test_sign_expand.py` passed. What problem did you have when using `qml.ctrl`?

```suggestion
    ops.append(qml.ctrl(op=qml.MultiRZ(theta, wires=list(active_wires)), control=controls[0]))
```

Reply:
I did try using it, it works, not sure why it didn't before, removed unnecessary code associated with it